Task Liverpool/src/test/java/com/encora/automation/test/BaseTest.java

```diff
@@ -1,32 +1,29 @@
 package com.encora.automation.test;
 
-import com.encora.automation.framework.DriverHandler;
-import com.encora.automation.pages.HomePage;
+import framework.DriverHandler;
 import org.openqa.selenium.WebDriver;
 import org.testng.annotations.AfterSuite;
-import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
 
 public class BaseTest {
 
-    protected HomePage homePage;
-
-    BaseTest(){
-        homePage = new HomePage(getDriver2());
-    }
-
-    private WebDriver getDriver2(){
+    protected WebDriver getDriver(){
         return DriverHandler.getWebDriver();
     }
 
-    @BeforeSuite
+    @BeforeTest
     protected void openLiverpoolURL(){
-        getDriver2().get("https://www.liverpool.com.mx/tienda/home");
+        getDriver().get("https://www.liverpool.com.mx/tienda/home");
```

Comment:
not all your test will start on homePage, move to HomeTest

Reply:
In this case, it does. I will leave it there so I don't have to add "getDriver().get("https://www.liverpool.com.mx/tienda/home");" to each of my 5 tests.